data/saas/config/slack_enterprise_config.yml

```diff
@@ -1,29 +1,27 @@
 saas_config:
   fides_key: <instance_fides_key>
-  name: Slack SaaS Config
-  type: slack
-  description: A sample schema representing the Slack connector for Fidesops
+  name: Slack Enterprise SaaS Config
+  type: slack_enterprise
+  description: A sample schema representing the Slack Enterprise connector for Fides
   version: 0.0.1
 
   connector_params:
     - name: domain
-    - name: access_token
+      default_value: slack.com
+    - name: user_oauth_token
+      label: User OAuth Token
 
   client_config:
     protocol: https
     host: <domain>
     authentication:
       strategy: bearer
       configuration:
-        token: <access_token>
+        token: <user_oauth_token>
 
   test_request:
     method: GET
     path: /api/api.test
-    query_params:
-      - name: email
-        value: test@test.com
-
 
   endpoints:
     - name: user
```

Comment:
just following up on [my comment](https://github.com/ethyca/fides/pull/1576#discussion_r1006147903) on the previous round of reviews - do we have good reasoning to exclude messages from at least our access requests?

Reply:
nevermind, i see in the PR description, just keeping this here for posterity

> Messages are not considered personal data per Slack's privacy guidelines

not pretending to be a legal expert but i'm struggling to see a convincing argument for this.

anyway, it's probably fair enough from our POV to defer to slack's policy. but do we have a current client looking to use this connector? if so, i'd say we should just run this by them to make sure we won't be scrambling to adjust if they have problems with the choice we've made here.